CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 tomland uses [PVP Versioning][1].
 The changelog is available [on GitHub][2].
 
+## 1.0.1
```

Comment:
Version bump up is correct :+1: Though let's add `Unreleased` tag for clarity
```suggestion
## Unreleased: 1.0.1
```

Reply:
Makes sense